akrivis-evaluator/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 # Jib extension
-quarkus.container-image.build=true
-quarkus.container-image.push=true
-quarkus.container-image.image=quay.io/lmolteni/akrivis-evaluator:latest
+# quarkus.container-image.build=true
+# quarkus.container-image.push=true
+# quarkus.container-image.image=quay.io/lmolteni/akrivis-evaluator:latest
```

Comment:
One thing I had to do is comment the container bits out, since it breaks the build.

Reply:
there should be some group account and maybe skip it for some builds with property/profile?